Automattic/jetpack
# Change `Inbox` menu name to `My Mailboxes`

Change menu name from `Inbox` to `My Mailboxes`. Rename relevant symbols, as well as route





## Proposed changes:

* Rename the `Inbox` menu name to `My Mailboxes`
* Rename all functions, comments and tests referring to the previous name
* Update tests as well

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Follow the directives specified [here](https://github.com/Automattic/jetpack/pull/32992#issuecomment-1715267300) to test in both Simple and Atomic sites.
* Go to your Calypso dashboard
* Confirm that the previous `Inbox` menu is now named `My Mailboxes` and pointing to `/mailboxes`.
